content/en/ddsql_reference/_index.md

```diff
@@ -125,8 +125,8 @@ The following SQL functions are supported. For Window function, see the separate
 | `TO_CHAR(timestamp t, string format)`            | string                                | Converts a timestamp to a string according to the given format.             |
 | `DATE_TRUNC(string unit, timestamp t)`           | timestamp                             | Truncates a timestamp to a specified precision based on the provided unit.  |
 | `REGEXP_LIKE(string s, pattern p)`               | Boolean                               | Evaluates whether a string matches a regular expression pattern.            |
-| `REGEXP_MATCH(string s, pattern p)`              | array of strings                      | Returns substrings of the first pattern match in the string.                |
-| `REGEXP_REPLACE(string s, pattern p, string replacement)`| string                            | Replaces the first occurrence of the pattern in a string with the replacement.        |
+| `REGEXP_MATCH(string s, pattern p)`              | array of strings                      | Returns substrings of the first pattern match in the string. Note: function is in Preview, request access via support.  |
+| `REGEXP_REPLACE(string s, pattern p, string replacement [, string flags ])`| string      | Replaces the first occurrence of the pattern in a string with the replacement. Optional `flags` argument, add `'g'` (global) to match all occurrences instead of first. Note: function is in Preview, request access via support.        |
```

Comment:
```suggestion
| `REGEXP_REPLACE(string s, pattern p, string replacement [, string flags ])`| string      | Replaces the first occurrence of the pattern in a string with the replacement. Optional `flags` argument, add `'g'` (global) to match all occurrences instead of first. Note: Function is in Preview; request access from Support.        |
```